e2e/e2e-test-parca.sh

```diff
@@ -0,0 +1,88 @@
+################################################################################
+#
+# This script is meant to be run from the root of this project
+#
+################################################################################
+
+#!/bin/bash
+
+set -euo pipefail
+
+function e2erun() {
+  minikube_cluster_up
+  deploy
+  # minikube_cluster_down genius, spin deploy and delete ugh
+}
+
+# Create local minikube cluster and deploys the dev env for parca and parca agent 
+function minikube_cluster_up() {
+  echo "Spinning up a parca dev cluster"
+  minikube start -p parca-dev \
```

Comment:
```suggestion
  minikube start -p parca-e2e \
```
nit: just to be super specific